src/lib/constants.js

```diff
@@ -46,3 +46,241 @@ export const RETURN_KEY = 'Enter';
 export const ESCAPE_KEY = 'Escape';
 export const COMMAND_KEY = 'Meta';
 export const CONTROL_KEY = 'Control';
+
+/**
+ * Trace type constants
+ */
+export const CHART_CLASS = {
+  POINTS_2D: 'POINTS_2D',
+  TERNARY: 'TERNARY',
+  PLOT_3D: 'PLOT_3D',
+  GEOGRAPHIC: 'GEOGRAPHIC',
+  MAPBOXGL: 'MAPBOXGL',
+  SINGLE: 'SINGLE',
+  FINANCIAL: 'FINANCIAL',
+  TABLE: 'TABLE',
+};
+
+// Layout specification for TraceTypeSelector.js
+export const CHART_CATEGORY = {
+  BUSINESS: 'BUSINESS',
+  SCIENCE: 'SCIENCE',
+  CHARTS_3D: 'CHARTS_3D',
+  FINANCIAL: 'FINANCIAL',
+  STATISTICS: 'STATISTICS',
+  MAPS: 'MAPS',
+};
+
```

Comment:
Could these structures be simpler, like:
```
export const CHART_CLASS = {
 GEOGRAPHIC: ['atlas', 'choropleth', 'scattermapbox']
}
```

CHART_CATEGORY is needed for the picker and CHART_CLASS for compatibility and disabling and enabling different plots. So I see why both are needed. But I don't see why we should repeat that whole object that lists all chart types..